### DISCUSSION TITLE
Aligning text labels on the start axis

### DISCUSSION THREAD

**Discussion by joseandroidengineer** [patrykandpatrick/vico#836] [Questions]
### Question

Hi! Love this library! I wanted to ask if there was a way to align the text labels on the startAxis when the horizontalLabelPosition is VerticalAxis.HorizontalLabelPosition.Outside. It aligns the text the way I want it to when I switch it to the Inside, but I want to keep the labels outside the chart. Is it possible? Do I have to add some margin or padding somewhere or mutate the startAxis label in some way? Let me know how this can be done, thank you so much!

### Vico version(s)

2.0.0 alpha.21

### UI framework(s)

Jetpack Compose

**Comment by Gowsky** [comment]
Hello, @joseandroidengineer. As per #778, this is a planned addition, and in the meantime, you can implement it by creating a `VerticalAxis` subclass and making some minor edits.
